sbpy/spectroscopy/core.py

```diff
@@ -51,9 +272,20 @@ def total_number(integrated_flux, frequency):
 
     not implemented
     """
```

Comment:
In this function you could use `bib.register` to register the article 2004come.book..391B when running this function.

Reply:
I will do this once the new bib.register functionalities are merged so I can rebase and apply this new bib functionality to our production rate functions.